sandbox-lincbrain.tfvars

```diff
@@ -0,0 +1,11 @@
+singleuser_image_repo = "dandiarchive/dandihub"
+region = "us-west-1"
+singleuser_image_tag = "latest"
+jupyterhub_domain = "hub-sandbox.lincbrain.org"
+jupyterhub_name = "sandbox-lincbrain"
+dandi_api_domain = "https://staging-api.lincbrain.org"
+admin_users = ["asmacdo", "dandibot", "satra", "aaronkanzer", "kabilar"]
+github_client_id = "fffb5efe40b01617b837"
+github_client_secret = "<something-i-am-storing-locally>"
```

Comment:
this should be set in an env.var rather than here

Reply:
https://github.com/lincbrain/linc-hub/pull/1#discussion_r1561743653